stream-chat-android-offline/src/test/java/io/getstream/chat/android/livedata/BaseDomainTest.kt

```diff
@@ -239,6 +253,7 @@ internal open class BaseDomainTest {
                 return 1000
             }
         }
+        chatDomainImpl.scope = testCoroutines.scope
```

Comment:
This should not be necessary, we're already mocking the dispatcher that's being used inside this scope via `DispatcherProvider` and `TestCoroutineRule`. Let's let the `ChatDomainImpl` create a real scope here, as it usually does. `scope` should actually be a `val` in that class, not sure why it's a `var` at the moment.